src/routes/results.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Button, StyleSheet, Text, View } from 'react-native';
+
+class Results extends React.Component {
+  render() { //I'm not sure how to get the spacings right for this. Right now it looks kinda ugly.
+    return (
+      <View
+        style={styles.search}>
+        <Text style={styles.textWithMargin}>That is a tin can!</Text>
+        <Text style={styles.textWithMargin}>Recycle it</Text> 
+        <Button
+          style={styles.textWithMargin}
+          title='Go To Landing'
+          onPress={() => this.props.navigation.navigate('Landing')}
+        />
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  search: {
+    flex: 1,
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    backgroundColor: '#fff',
+  },
+  textWithMargin: {
```

Comment:
It's a better practice to name your styles after the element they are connected to, not what they do.
When I get time to do the designs, I'm going to stick pretty close to this pattern http://www.seattle.gov/util/MyServices/WhereDoesItGo/GlassCeramics/BrokenGlass/index.htm 
So I'm thinking we will need a style for the result header, one for the action (eg Recycle), and one for the description text.